### PR TITLE
[v6r19] pilot: CPUTime is discovered always at runtime

### PR DESCRIPTION
**Pull request by fstagni**
This is a fix for pilots in v6r19p14

BEGINRELEASENOTES

*WMS
FIX: SiteDirector won't set CPUTime of the pilot
FIX: convert MaxRAM inside the pilots to int

*RSS
FIX: SummarizeLogsAgent: comparison bug fix
FIX: Fixed sites synchronizer

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/PilotAgent/pilotCommands.py [hunk 1]
```diff
@@ -504,9 +504,13 @@ def execute( self ):
     self.cfg.append(
         '-o "/Resources/Computing/CEDefaults/NumberOfProcessors=%d"' % numberOfProcessors)
 
-    maxRAM = self.pp.queueParameters.get('MaxRAM', maxRAM)
+    maxRAM = int(self.pp.queueParameters.get('MaxRAM', maxRAM))
```
Comment (andresailer) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: the cast to `int` should be dropped here, as it is done again below and there with try/except

File: WorkloadManagementSystem/PilotAgent/pilotCommands.py [hunk 2]
```diff
@@ -504,9 +504,13 @@ def execute( self ):
...
     if maxRAM:
-      self.cfg.append( '-o "/Resources/Computing/CEDefaults/MaxRAM=%d"' % maxRAM )
+      try:
+        self.cfg.append(
+            '-o "/Resources/Computing/CEDefaults/MaxRAM=%d"' % int(maxRAM))
+      except ValueError:
+        self.log.warn("MaxRAM is not an integer, won't filling it")
```
Comment (andresailer) [inline WorkloadManagementSystem/PilotAgent/pilotCommands.py]: "`won't be filling it`" or something without a quote for easier grepping... "`will not fill it`"...

**Comment by fstagni** [comment]
This should be fine, please go ahead.

**Comment by fstagni** [comment]
I profited for another fix in RSS.

**Pull request merged**